src/pages/Components/Notification/code.md

```diff
@@ -7,26 +7,40 @@ order: 2
 
 ```css
 @import 'settings-tools/_all-settings';
-@import '...';
+@import 'components/_c.notification';
 ```
 
 ## Basic usage
 
-Apply the classes...
+Wrap a `h3`, `p` and a `a` or `button` tags in a `div` and apply the `mc-notification` class:
 
 ```html
-<div class="class">Label</div>
+  <div class="mc-notification">
+    <h3>Notification title</h3>
+    <p>
+      Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa.
+    </p>
+    <a href="#">Link</a>
+  </div>
 ```
```

Comment:
You have to add classes on your elements.

Reply:
Gael decided that the adoption of the `role="alert` depends on the context, if the "Notification" is being used to alert the user about a result of your action. So the "Notification" should play as an "alert" indeed.